style.css

```diff
@@ -490,3 +508,429 @@ input[type="submit"]:disabled {
   padding-bottom: 0;
   background: var(--line);
 }
+
+/* all the display none for mobile */
+nav > :nth-child(3) {
+  display: none;
+}
+
+.imgsection > :nth-child(2) {
+  display: none;
+}
+
+.cartt2 {
+  display: none;
+}
+
+form#f2 {
+  display: none;
+}
+
+@media only screen and (min-width: 790px) {
+  body {
+    padding: 0;
+    margin: 0;
+  }
+
+  /* The navigation selector */
+
+  /* all the display none for desktop */
+  nav > :nth-child(2) {
+    display: none;
+  }
+
+  #headline > h2 {
+    padding-left: 4.5rem;
+  }
+
+  #about > h2,
+  p {
+    padding-left: -0.5rem;
+  }
+
+  #contact > h2 {
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    letter-spacing: 0;
+    text-align: left;
+    height: auto;
+    width: 28rem;
+    margin-left: -9rem;
+    padding: 2rem;
+  }
+
+  #work > .first-box > .headtext > h2 {
+    border: none;
+    padding-left: 4.5rem;
+  }
+
+  nav > :nth-child(3) {
+    display: flex;
+  }
+
+  .imgsection > :nth-child(2) {
+    display: flex;
+    min-width: 42.5em;
+    height: 26rem;
+  }
+
+  .imgsection > :nth-child(1) {
+    display: none;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: auto;
+    padding: 1.8rem 6rem 1rem 6rem;
+    background: var(--navcolor);
+  }
+
+  nav > ul > li {
+    list-style-type: none;
+  }
+
+  nav > ul > li > a {
+    list-style-type: none;
+    text-decoration: none;
+    color: var(--txtcolor);
+    font-family: 'Inter', sans-serif;
+    font-size: 1rem;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: center;
+    padding-left: 2rem;
+  }
+
+  .carttt {
+    padding: 0;
+    margin: 0;
+    display: none;
+  }
+
+  #headline {
+    background-image: url(Resources/Images/Illustration-header.png);
+    background-position: top right;
+    background-size: 30rem;
+    height: 30rem;
+    padding-left: 1rem;
+  }
+
+  .imgsection > p {
+    max-width: 22rem;
+  }
+
+  #headline > p {
+    color: var(--txtcolor);
+    padding-top: 1rem;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 1rem;
+    line-height: 24px;
+    width: 34em;
+    flex-grow: 0;
+    margin: 0 0;
+    padding-left: 4.5rem;
+  }
+
+  #headline > ul {
+    position: relative;
+    float: left;
+    top: -14rem;
+    flex-direction: column;
+  }
+
+  .headtext {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
```

Comment:
- [ ] Great job so far, to be consistent with the design please remove the space after `My Recent Works` and also on the right the line ending should also be where the text and cards end.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/168446734-c81ab9d1-eb14-4c6a-95da-2be6160c4c25.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/168446756-7e998893-b6a2-4045-871a-7761cce7ecc6.png)